lib/tasks/data_migration.rake

```diff
@@ -10,6 +10,22 @@ namespace :data_migration do
     SubscriberListMover.new(**args).call
   end
 
+  desc "Rename an alert type and all combinations of subscriptions to it"
+  task :rename_alert_subscription_lists, %i[from_slug to_slug] => :environment do |_t, args|
+    SubscriberList.where("tags->'alert_type' IS NOT NULL").all.each do |list|
+      next unless list.tags[:alert_type][:any].include? args[:from_slug]
+
+      new_tags = (list.tags[:alert_type][:any] - [args[:from_slug]] + [args[:to_slug]]).uniq
+
+      if (new_list = SubscriberList.where("tags->'alert_type' IS NOT NULL").find_all { |l| l.tags[:alert_type][:any].sort == new_tags.sort }.first)
```

Comment:
I'm guess there's the potential for the `new_list` and the `old_list` to end up the same? For example if there's a subscriber list for alerts types that aren't related to the old or new slug. Do we need a check for this?

Reply:
I'm not entirely sure of the circumstances this would occur in, but it can't hurt to make sure.